### PR TITLE
DEVDOCS-2227: Section update

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2227](https://jira.bigcommerce.com/browse/DEVDOCS-2227)

## What changed?
* added html & handlebars formatting
* added a link to cornerstone in guthub

### CODE REVIEW

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 1]
```diff
@@ -36,6 +36,37 @@ In this article, we are going to review all the components that make a widget. T
...
+### Global regions
+
+Global regions are special regions you can use to place and manage content sitewide. Widgets placed in a global region will appear on the storefront pages where that region exists. This makes global regions useful for displaying high-priority information, such as special promotion advertising banners, across multiple storefront pages.
+
+ ![Global Region](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/widgets-overview-03.png "Global Region")
+
+To create a global region, add the `--global` suffix to the region name as shown in the following example:
+
+```handlebars
+
```
Comment (aglensmith) [inline docs/api-docs/storefront/widgets/widgets-overview.md:48]:
```suggestion
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 2]
```diff
@@ -36,6 +36,37 @@ In this article, we are going to review all the components that make a widget. T
...
+{{{region name=”header_bottom--global”}}}
+
```
Comment (aglensmith) [inline docs/api-docs/storefront/widgets/widgets-overview.md:50]:
```suggestion
```

File: docs/api-docs/storefront/widgets/widgets-overview.md [hunk 3]
```diff
@@ -36,6 +36,37 @@ In this article, we are going to review all the components that make a widget. T
...
+```
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
```
Comment (aglensmith) [inline docs/api-docs/storefront/widgets/widgets-overview.md:56]:
```suggestion
```

**Review by aglensmith — CHANGES_REQUESTED** [review]

**Comment by uwikaiddi** [comment]
@ektravel, please resolve the merge conflicts for this first. I'' merge after.

**Pull request closed without merging**